pkg/question_generator/questiongenerator.go

```diff
@@ -0,0 +1,19 @@
+package question_generator
+
+import (
+	"math/rand"
+	"net/http"
+
+	"github.com/crisBrunette/questioncicas/internal/database"
+)
+
+var RamdonQuestion string
+
+func Generator(req *http.Request) string {
+	keys := make([]string, 0, len(database.QuestionAnswer)) // len = 0, cap = 5(database.QuestionAnswer)
+	for k := range database.QuestionAnswer {
+		keys = append(keys, k)
+	}
+	RamdonQuestion = keys[rand.Intn(len(keys))]
```

Comment:
```suggestion
	RandomQuestion = keys[rand.Intn(len(keys))]
```
And also it needs to be changed in other places, when we work on the CI we will be able to check that this code builds correctly on every PR so it's not possible to merge something that fails, but for now... that's not there, so remember to change the var to be `rand.Intn` this